### PR TITLE
feat: create files methods in files package to upload, findone and delete files

### PR DESCRIPTION
**Pull request by robertotorres-dev**
…lete files

### CODE REVIEW

File: packages/files/src/useCases/files.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    throw boom.notFound(
+      `[files:finOne]: File not found ${tipoDocumento} with entidadId: ${entidadId}`,
+    );
+  }
+
+  return file;
+};
+
+const uploadFile = (createQuery, updateQuery, findOneByParamsQuery) => async (dataFile, file) => {
+  const { tipoEntidad, entidadId, tipoDocumento } = dataFile;
```
Comment (hecttoressau) [inline packages/files/src/useCases/files.js:39]: remember to rename tipoEntidad to tipoEntidadId

File: packages/files/src/useCases/files.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    fileSaved = await updateQuery(previousFile.id, data);
+  } else {
+    fileSaved = await createQuery(data);
+  }
+
+  return fileSaved;
+};
+
+const deleteByParams = (deleteQuery, findOneByParamsQuery) => async (dataFile) => {
+  const { tipoEntidad, entidadId, tipoDocumento } = dataFile;
```
Comment (hecttoressau) [inline packages/files/src/useCases/files.js:80]: remember to rename tipoEntidad to tipoEntidadId

File: packages/files/src/useCases/files.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
+/* eslint-disable no-undef */
+const boom = require('@hapi/boom');
+const { TIPO_ENTIDAD, TIPO_DOCUMENTO } = require('../utils/constants');
+const { writeBus, deleteBus } = require('../utils/manageFiles');
+
+const findOneByParams = (findOneByParamsQuery) => async (dataFile) => {
+  const { tipoEntidad, entidadId, tipoDocumento } = dataFile;
```
[the remark this answers is not in the document]
Comment (csiges) [reply packages/files/src/useCases/files.js:7]: I rename the data to send tipoEntidadId and tipoDocumentoId at the database
But in this part, I recive a tipoEntidadName and tipoDocuementoName
Please check the new changes that I did.
Thank you!

File: packages/files/src/useCases/files.js (hunk 1 above)
Comment (csiges) [reply packages/files/src/useCases/files.js:39]: I renamed the data to send tipoEntidadId and tipoDocumentoId at the database
But in this part, I recive a tipoEntidadName and tipoDocuementoName
Please check the new changes that I did.
Thank you!

File: packages/files/src/useCases/files.js (hunk 2 above)
Comment (csiges) [reply packages/files/src/useCases/files.js:80]: I renamed the data to send tipoEntidadId and tipoDocumentoId at the database
But in this part, I recive a tipoEntidadName and tipoDocuementoName
Please check the new changes that I did.
Thank you!

**Review by hecttoressau — APPROVED** [review]
LGTM

**Pull request merged**